server/services/qBittorrent/types/QBittorrentTorrentsMethods.ts

```diff
@@ -123,6 +123,8 @@ export interface QBittorrentTorrentsAddOptions {
   skip_checking?: boolean;
   // Add torrents in the paused state. Possible values are true, false (default)
   paused?: boolean;
+  // Add torrents in the stopped state (using webapiVersion v2.11.0 or later). Possible values are true, false (default)
+  stopped?: boolean;
```

Comment:
isn't “add stop” has same meaning as ”add paused“？

Reply:
It's to take care of the api changes with qbit 5 where it's now using "stopped" instead of "paused". So in order to fit both earlier and later versions of the api both should be there. Makes sense?

Reply:
I understand there is a api change in qb v5, but in my in my understanding it rename from "paused" to "stopped", and they have actually same meaning, correct?

So we I think we do not need to handle it outside clientGatewayService.ts and clientRequestManager.ts

Reply:
Forgive me if im wrong but doesn't "stopped" need to be a valid key when creating the options (based on QBittorrentTorrentsAddOptions interface) in ` await this.clientRequestManager.torrentsAddFiles(fileBuffers, {.....` line 88 in clientGateWayService ?

Reply:
OK, I understand what you are doing here.